src/wp-login.php

```diff
@@ -210,10 +210,16 @@ function login_header( $title = null, $message = '', $wp_error = null ) {
 	 * @since 4.6.0
 	 */
 	do_action( 'login_header' );
-
+	?>
+	<?php
+	if ( 'confirm_admin_email' !== $action ) :
```

Comment:
```suggestion
	if ( 'confirm_admin_email' !== $action && ! empty( $title ) ) :
```

This might need to check for an empty `$title` as well. The [successful interim login](https://github.com/WordPress/wordpress-develop/blob/bd9e141608a0879ee1093091b4b0031a28b20583/src/wp-login.php#L1365) uses an empty string in `login_header()`.